markdown/usage/troubleshooting.md

```diff
@@ -99,6 +154,101 @@ Specifying the version of the run you are using is highly recommended, as it hel
+
+Once saved, we can then modify your original Nextflow run command:
+
+```bash
+nextflow run nf-core/<PIPELINE> -c /<path>/<to>/custom_resources.conf -profile big_data,<original>,<profiles> <...>
+```
+
+Where we have added `-c` to specify which file to use for the custom profiles, and then added the `big_data` profile to the original profiles you were using.
+
+:warning: it's important that big_data comes first, to ensure it overwrites any parameters set in the subsequent profiles!
+
+## Crashed pipeline with an error but Nextflow is still running
+
+If this happens, you can either wait until all other already running jobs to safely finish, or if Nextflow _still_ does not stop press `ctrl + c` on your keyboard (or equivalent) to stop the Nextflow run.
+
+> :warning: if you do this, and do not plan to fix the run make sure to delete the output folder. Otherwise you may end up a lot of large intermediate files being left! You can clean a Nextflow run of all intermediate files with `nextflow clean -f -k` or delete the `work/` directory.
+
+### I specified a step of a pipeline to run and it wasn't executed
+
+Possible options:
+
+1. Check there if you have a typo in the parameter name. Nextflow _does not_ check for this
+2. Check that an upstream step/process was turned on (if a step/process requires the output of an earlier process, it will not be activated unless it receives the output of that process)
+
+## My pipeline update doesn't seem to do anything
+
+To download a new version of a pipeline, you can use the following, replacing `<VERSION>` to the corresponding version.
+
+For example for nf-core/eager:
+
+```bash
+nextflow pull nf-core/eager -r <VERSION>
+```
+
+However, in very rare cases, minor fixes to a version will be pushed out without a version number bump. This can confuse Nextflow slightly, as it thinks you already have the 'broken' version from your original pipeline download.
+
+If when running the pipeline you don't see any changes in the fixed version when running it, you can try removing your nextflow EAGER cache typically stored in your home directory with (again for eager):
+
+```bash
+rm -r ~/.nextflow/assets/nf-core/eager
+```
+
+And re-pull the pipeline with the command above. This will install a fresh version of the version with the fixes.
+
+## Unable to acquire lock error
+
+Errors like the following:
+
+```bash
+Unable to acquire lock on session with ID 84333844-66e3-4846-a664-b446d070f775
+```
+
+Normally suggest a previous Nextflow run (on the same folder) was not cleanly killed by a user (e.g. using ctrl + z to hard kill a crashed run).
+
+To fix this, you must clean the entirety of the output directory (including output files) e.g. with `rm -r <output_dir>/* <output_dir>/.*` and re-running from scratch.
```

Comment:
`<output dir>` is ambiguous. Better to specifically say the work directory I think.

Also, why not just `rm -r <output_dir>` instead of `rm -r <output_dir>/* <output_dir>/.*`?